tracer/build/_build/BuildVariables.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using Nuke.Common;
+using Nuke.Common.IO;
+
+public static class BuildVariables
+{
+    public static void AddDebuggerEnvironmentVariables(this Dictionary<string, string> envVars, AbsolutePath tracerHomeDirectory)
+    {
+        envVars.AddContinuousProfilerEnvironmentVariables(tracerHomeDirectory);
+        envVars.Add("DD_DEBUGGER_ENABLED", "1");
+    }
+
+    public static void AddContinuousProfilerEnvironmentVariables(this Dictionary<string, string> envVars, AbsolutePath tracerHomeDirectory)
```

Comment:
I've added `AddTracerEnvironmentVariables`, which adds tracer env variables.
`AddDebuggerEnvironmentVariables` and `AddDebuggerEnvironmentVariables` will call it for now.
Anyway, it would be much easier for profiler and debugger teams to change it, according to their needs.